tests/cross/src/test/scala/tests/pc/AutoImportsSuite.scala

```diff
@@ -94,7 +94,7 @@ class AutoImportsSuite extends BaseAutoImportsSuite {
        |}
        |}
        |""".stripMargin,
-    filename = "A.sc",
+    filename = "A.sc.scala",
```

Comment:
This test seems to be failing on scala 3, because the import gets inserted at the top of the file